pkg/util/utils.go

```diff
@@ -334,3 +334,11 @@ func GetDefaultStoreOptions() (storage.StoreOptions, string, error) {
 	}
 	return storageOpts, volumePath, nil
 }
+
+// StorageConfigFile returns the path to the storage config file used
+func StorageConfigFile() string {
+	if rootless.IsRootless() {
+		return filepath.Join(os.Getenv("HOME"), ".config/containers/storage.conf")
```

Comment:
Can we make libpod use this function to decide what path to use when rootless? I don't want to have this change in one place but not the other

Reply:
It is using this function. just above.